Mathlib/Data/Nat/Digits.lean

```diff
@@ -637,6 +637,22 @@ theorem ofDigits_mod (b k : ℕ) (L : List ℕ) : ofDigits b L % k = ofDigits (b
   ofDigits_modEq b k L
 #align nat.of_digits_mod Nat.ofDigits_mod
 
+@[simp]
+theorem ofDigits_empty (b : ℕ) : ofDigits b [] % b = 0 := rfl
```

Comment:
```suggestion
theorem ofDigits_nil (b : ℕ) : ofDigits b [] % b = 0 := rfl
```
Sorry, I know that I was the one who named these lemmas, but I think that `nil` is the standard name, not `head`.